src/App.css

```diff
@@ -144,13 +143,11 @@ main > h1 + * {
   height: 100%;
 }
 
-footer {
+footer .btn-container {
```

Comment:
@jssckbl  Is it possible?, add a little space between the first  & second row. 
![image](https://user-images.githubusercontent.com/8689897/131945204-8235f3bb-2b37-48cf-a5c0-1fc994e8da93.png)

Reply:
Possible solution 

![image](https://user-images.githubusercontent.com/8689897/131945325-691b723f-a61b-4f2f-9e92-71a6dcc6163a.png)

of course without backgroud-color 😄

Reply:
For sure!

Reply:
Added 0.5rem.